monitors/BalanceMonitor.js

```diff
@@ -52,40 +52,40 @@ class BalanceMonitor {
     // check if their collateral, synthetic or ether balance is below a given threshold. If it is, then
     // send a winston event. The message structure is defined with the `_createLowBalanceMrkdwn` formatter.
     for (let bot of this.botsToMonitor) {
-      if (this._ltThreshold(this.client.getCollateralBalance(bot.address), bot.collateralThreshold)) {
```

Comment:
This `_ltThreshold` was previously used to deal with the `null` case if a balance had yet to be resolved. As this is now async this is no longer needed.